packages/go/ein/ad.go

```diff
@@ -34,9 +35,13 @@ func ConvertSessionObject(session Session) IngestibleSession {
 }
 
 func ConvertObjectToNode(item IngestBase, itemType graph.Kind) IngestibleNode {
+	properties := item.Properties
+	properties[ad.IsACLProtected.String()] = item.IsACLProtected
```

Comment:
This is probably ok, but I would still prefer to add these properties during collection.

Reply:
Ripped this out 👍